Temporary patch added for log to file failed validation in controller for success transaction to help with debugging.

diff --git a/Controller/Transaction/Success.php b/Controller/Transaction/Success.php
index d85469c..e0f013b 100644
--- a/Controller/Transaction/Success.php
+++ b/Controller/Transaction/Success.php
@@ -43,6 +43,13 @@ class Success extends \Wallee\Payment\Controller\Transaction
      */
     private $transactionService;

+    // BEGIN PATCH
+    /**
+     * @var \Creativestyle\CustomizationIpetPayment\Logger\Logger
+     */
+    protected $logger;
+    // END PATCH
+
     /**
      *
      * @param Context $context
@@ -58,6 +65,10 @@ class Success extends \Wallee\Payment\Controller\Transaction
         $this->checkoutSession = $checkoutSession;
         $this->successValidator = $successValidator;
         $this->transactionService = $transactionService;
+
+        // BEGIN PATCH
+        $this->logger = \Magento\Framework\App\ObjectManager::getInstance()->get(\Creativestyle\CustomizationIpetPayment\Logger\Logger::class);
+        // END PATCH
     }

     public function execute()
@@ -72,6 +83,15 @@ class Success extends \Wallee\Payment\Controller\Transaction
             ], 5);

         if (! $this->successValidator->isValid()) {
+            // BEGIN PATCH
+            // Temporary patch added for log to file failed validation in controller for success transaction to help with debugging.
+            $params = $this->getRequest()->getParams();
+            $this->logger->warning(sprintf(
+                'Validation problem in Wallee\Payment\Controller\Transaction\Success - params: %s, php errors: %s',
+                json_encode($params),
+                json_encode(error_get_last())
+            ));
+            // END PATCH
             $this->messageManager->addErrorMessage(
                 \__(
                     'There seems to have been a problem with your order. ' .
